### PR TITLE
feat: Add changelog for additional plugin support for consumer groups in Konnect

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

- Along with Gateway 3.6 support, Konnect now supports several more plugins for consumer groups. This adds a changelog entry for this change.
 

DOCU-3600
### Testing instructions

Preview link: https://deploy-preview-6812--kongdocs.netlify.app/konnect/updates/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/updates.md
Comment (cloudjumpercat) [inline app/konnect/updates.md:22]: @hl782 Hi! I pulled this list of supported plugins from the brief, and I just wanted to verify with you that this list is correct. Thanks!

File: app/konnect/updates.md
Comment (cloudjumpercat) [inline app/konnect/updates.md:17]: ❗ For reviewers: Do we also need to edit these plugin pages with any Konnect Paid/Premium badges?

**Comment by cloudjumpercat** [comment]
❗ For reviewers: I added a "do not merge" badge because I wasn't sure how we should handle Konnect release merges now that `dev` is available.

File: app/konnect/updates.md
Comment (cloudjumpercat) [inline app/konnect/updates.md:24]:
```suggestion
```

**Review by Guaris — APPROVED** [review]

**Comment by cloudjumpercat** [comment]
Hi @hl782 ! The PR for the Konnect additional plugin support changelog is now approved and ready for you to merge once the feature goes GA! 

Here’s a link to the preview of the changelog in the docs: [Kong Konnect Updates | Kong Docs](https://deploy-preview-6812--kongdocs.netlify.app/konnect/updates/) 

Let me know if you have any questions!

**Pull request merged**
